infrastructures/infrastructure-common/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/util/LinuxInitScriptGenerator.java

```diff
@@ -36,6 +36,8 @@
 
 public class LinuxInitScriptGenerator {
 
+    public static final String DEFAULT_SUFFIX_RM_TO_NODEJAR_URL = ":8080/rest/node.jar";
```

Comment:
Adding the url suffix as a vriable is a good idea, but it would be better to add it in the properties file then read it (as we do with the jre commands).

Reply:
Done. However, this request has required intrusive changes in the code. Please evaluate if this request is still pertinent to be maintained. Otherwise, I can just revert my commit.